docs/source/target-shell.rst

```diff
@@ -0,0 +1,65 @@
+target-shell
```

Comment:
This page is in my opinion missing a description of the "registry" command in target-shell. This is definitely an option which could be a massive timesaver